### PR TITLE
Update menu order and fix link

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
Ordered in this way makes more sense.
Fixed a 404 link

### CODE REVIEW

**Comment by yahiheb** [comment]
IMO the order is good as it is, address reuse should be number one on that list.

**Comment by RiccardoMasutti** [comment]
@yahiheb I'd start with how Coins/input outputs works

**Comment by MaxHillebrand** [comment]
I agree with @RiccardoMasutti, coins are so fundamental that they should be before address reuse [meaning to * **coins** *  have the same address]

**Comment by yahiheb** [comment]
> [meaning to * coins * have the same address

How do you avoid having two coins in the same address if you don't know about address reuse?

From our documentation:
> The first rule of Bitcoin privacy is never reuse addresses!

> The first rule of Bitcoin privacy:
> 
> > Never reuse addresses!
> 
> The second rule of Bitcoin privacy:
> 
> > NEVER reuse addresses!!

File: docs/why-wasabi/README.md [hunk 1]
```diff
@@ -21,9 +21,10 @@ And why Wasabi can make Bitcoin an even better tool of self-defense by default.
 - [Why Financial Privacy is Especially Important](/why-wasabi/WhyFinancialPrivacy.md)
 - [CoinJoin Legal Concern](/why-wasabi/LegalConcern.md)
 
-### Privacy in Bitcoin
-- [Address Reuse](/why-wasabi/AddressReuse.md)
+#### Privacy in Bitcoin
```
Comment (yahiheb) [inline docs/why-wasabi/README.md:24]:
```suggestion
### Privacy in Bitcoin
```

**Pull request merged**